src/coreclr/hosts/unixcoreruncommon/coreruncommon.cpp

```diff
@@ -16,7 +16,11 @@
 #include <string>
 #include <string.h>
 #include <sys/stat.h>
-#ifdef HAVE_SYS_SYSCTL_H
```

Comment:
You can try to let cmake generate preprocessed src/coreclr/hosts/unixcoreruncommon/coreruncommon.cpp and see where the sysctl function comes from. Just go to the `bin/obj/Linux.x64.Debug/src/coreclr/hosts/unixcoreruncommon` (replace Linux.x64.Debug with whatever you have) and run `make coreruncommon.cpp.i`. Then look at the generated file to see where the sysctl came from.